lib/booking/class/booking.dart

```diff
@@ -50,7 +50,7 @@ class Booking {
           ? Applicant.fromJson(json["applicant"])
           : Applicant.empty().copyWith(id: json["applicant_id"]),
     );
-    return booking.copyWith(end: getTrueEnd(booking));
+    return booking.copyWith();
```

Comment:
Can't you just return the booking, instead of creating a copy that is the object

Reply:
Yes I'll remove the copyWith